File: Java/1048.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);

        double sal = leitor.nextDouble();

        if (sal <= 400) {
            System.out.printf("Novo salario: %.2f\n", (sal * 1.15));
            System.out.printf("Reajuste ganho: %.2f\n", (sal * 0.15));
            System.out.println("Em percentual: 15 %");
        }
        
        if (sal > 400 && sal <= 800) {
            System.out.printf("Novo salario: %.2f\n", (sal * 1.12));
            System.out.printf("Reajuste ganho: %.2f\n", (sal * 0.12));
            System.out.println("Em percentual: 12 %");
        }
        
        if (sal > 800 && sal <= 1200) {
            System.out.printf("Novo salario: %.2f\n", (sal * 1.10));
            System.out.printf("Reajuste ganho: %.2f\n", (sal * 0.10));
            System.out.println("Em percentual: 10 %");
        }
        
        if (sal > 1200 & sal <= 2000) {
            System.out.printf("Novo salario: %.2f\n", (sal * 1.07));
            System.out.printf("Reajuste ganho: %.2f\n", (sal * 0.07));
            System.out.println("Em percentual: 7 %");
        }
        
        if (sal > 2000) {
            System.out.printf("Novo salario: %.2f\n", (sal * 1.04));
            System.out.printf("Reajuste ganho: %.2f\n", (sal * 0.04));
            System.out.println("Em percentual: 4 %");
        }
    }
 
}

File: Python/1048.py
# -*- coding: utf-8 -*-
sal = float(input())

if(sal <= 400):
    print("Novo salario: %.2f" % (sal * 1.15))
    print("Reajuste ganho: %.2f" % (sal * 0.15))
    print("Em percentual: 15 %")
    
if(sal > 400 and sal <= 800):
    print("Novo salario: %.2f" % (sal * 1.12))
    print("Reajuste ganho: %.2f" % (sal * 0.12))
    print("Em percentual: 12 %")
    
if(sal > 800 and sal <= 1200):
    print("Novo salario: %.2f" % (sal * 1.10))
    print("Reajuste ganho: %.2f" % (sal * 0.10))
    print("Em percentual: 10 %")

if(sal > 1200 and sal <= 2000):
    print("Novo salario: %.2f" % (sal * 1.07))
    print("Reajuste ganho: %.2f" % (sal * 0.07))
    print("Em percentual: 7 %")

if(sal > 2000):
    print("Novo salario: %.2f" % (sal * 1.04))
    print("Reajuste ganho: %.2f" % (sal * 0.04))
    print("Em percentual: 4 %")